src/modules/projects/ProjectHome.tsx

```diff
@@ -85,18 +89,20 @@ export function ProjectHome({ section, children }: Props) {
                       ))
                     }
                   />
-                  <OverflowMenuItem
-                    itemText="Archive"
-                    isDelete
-                    onClick={() =>
-                      openModal((props) => (
-                        <ArchiveConfirmationModal
-                          {...props}
-                          project={project}
-                        />
-                      ))
-                    }
-                  />
+                  {project.id !== defaultProject && (
+                    <OverflowMenuItem
+                      itemText="Archive"
```

Comment:
Or have it disabled with a tooltip why, but I guess this is also fine.

Reply:
I couldn't find that it would be possible to add a tooltip to disabled OverflowMenuItem. Carbon doesn't seem to support it.